examples/nextjs/pages/_document.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import Document, { Head, Main, NextScript } from 'next/document';
+import { ServerStyleSheet } from 'styled-components';
+import { IdManager } from '@zendeskgarden/react-selection';
+
+import BedrockStyles from '@zendeskgarden/css-bedrock/dist/index.css';
+import ChromeStyles from '@zendeskgarden/react-chrome/dist/styles.css';
+import ButtonStyles from '@zendeskgarden/react-buttons/dist/styles.css';
+import MenuStyles from '@zendeskgarden/react-menus/dist/styles.css';
```

Comment:
I could be wrong but I think the idiomatic way to add styles to a nextjs app is via the `withCSS` plugin:

This example should have a `next.config.js` file in the root.

```js
const withCSS = require('@zeit/next-css');

module.exports = withCSS();
```

Then in `_document.js` you can just add `<link rel="stylesheet" href="/_next/static/style.css" />`

Then in your index or layout file you can just do nameless imports for the styles needed.

Reply:
I stole this from one of the examples in their repo. 

I think the one you linked to is better though. Will implement in that style.